build/releaseBuild.yml

```diff
@@ -0,0 +1,223 @@
+trigger: none
+pr: none
+
+name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+# Create a daily midnight build for release builds on master to ensure our release builds function
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - dev
+
+variables:
+  - group: Middleware
+  - name: BuildPlatform
+    value: 'any cpu'
+  - name: BuildConfiguration
+    value: 'Release'
+
+jobs:
+- job: build
+  pool:
+    name: MwWilson1EsHostedPool
```

Comment:
I ran into an issue before because the host was an older version of Windows, and I am trying to figure out how that is set in here. Are the agent host specifications in MwWilson1EsHostedPool?

Reply:
Yes, it's specified in this new 1ES pool that @jmprieur created.